common-practices-tools/software-and-support/mural.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Mural
+---
+
+# Mural
+
+The following is a general overview of Mural, but each project may have specific setups or workflows that differ. If you have any questions, please ask in either the [\#mural](https://civicactions.slack.com/archives/C06PASS8S6M) or in [\#how-we-work](https://civicactions.slack.com/archives/C02HFM4KR) slack channels.
+
+## About Mural
+
+CivicActions uses [Mural](https://app.mural.co/t/civicactions3117) as a digital workspace for visual collaboration. It's particularly useful for brainstorming, planning, and design thinking projects. Mural allows teams to work together in real-time, regardless of their geographical locations, and is a powerful tool for enhancing collaborative efforts.
+
+<div align="center">
+  <a href="https://app.mural.co/t/civicactions3117">
+    <img src="https://img.shields.io/badge/Mural_Account-0F6460?style=for-the-badge&logo=mural" alt="CivicActions Mural Account">
+  </a>
+  <a href="https://civicactions.slack.com/archives/C06PASS8S6M">
+    <img src="https://img.shields.io/badge/Slack_Channel-D83933?style=for-the-badge&logo=slack" alt="#mural Slack Channel">
+  </a><br><br>
+</div>
```

Comment:
Can we use markdown here? We have ability to use https://squidfunk.github.io/mkdocs-material/reference/grids/ if needed.

Reply:
Also capture the images as assets instead of linking to outside versions? That is minor though